### DISCUSSION TITLE
Test PRs with nix

### DISCUSSION THREAD

**Discussion by tcoopman** [helix-editor/helix#9589] [Q&A]
I'm using nix to install Helix and I'd like to tests PRs every now and then.
I'm wondering if anyone on here is doing the same and has some suggestions on how to to this?

I've found this discussion but I don't think this is the same question: https://github.com/helix-editor/helix/discussions/9385

**Comment by TornaxO7** [comment]
If you just want to try the PRs, simply switch to them and run `cargo run`?

**Reply by tcoopman** [reply]
That could work but I like my setup/config to be in nix.
I have 3 PCs that I switch between, manually handling all of this becomes a bit of a chore.

**Reply by TornaxO7** [reply]
are you using nix flakes? If yes, than you can simple add an input entry for helix and use the url of the suitable PR, add it to your packages and then it's installed.

In my case for example, I'm always using [the master branch](https://github.com/TornaxO7/dotfiles/blob/nixos/flake.nix#L13) of helix.
